### ISSUE TITLE
iOS app - out of memory

### ISSUE THREAD

**Issue by romkij** [artoolkitx/arunityx#7]
Hello!
I can't run app to iOS with Unity3D, I get error in Xcode, when running app on device - "Out of memory".

Thank you!

```### Surface No.1 ###
2018-08-23 12:54:54.065377+0300 arbankx[10737:1217974] [info]   Read ImageSet.
2018-08-23 12:54:54.066458+0300 arbankx[10737:1217974] [info] Imageset contains 7 images.
2018-08-23 12:54:54.067365+0300 arbankx[10737:1217974] [error] Error reading JPEG file.
2018-08-23 12:54:54.067404+0300 arbankx[10737:1217974] [warning] Falling back to reading '/var/containers/Bundle/Application/A488DC6D-D322-4A9D-A134-FEE93F3F3891/arbankx.app/Data/Raw/gibraltar.iset' in ARToolKit v4.x format.
arbankx(10737,0x1b5c54b40) malloc: *** mach_vm_map(size=17833407189369618432) failed (error code=3)
*** error: can't allocate region
*** set a breakpoint in malloc_error_break to debug
2018-08-23 12:54:54.068012+0300 arbankx[10737:1217974] [error] Out of memory!!```

**Comment by chuen1118** [comment]
This might be the same issue as my post [here](https://github.com/artoolkitx/artoolkitx/issues/47#issue-348192698) and my [solution](https://github.com/artoolkitx/artoolkitx/issues/47#issuecomment-411791255).

**Comment by philip-lamb** [comment]
This was a bug which should be fixed in artoolkitX v1.0.5.
